org-roam-node.el

```diff
@@ -283,18 +294,61 @@ FILTER-FN is the name of a function to apply on the candidates
 which takes as its argument an alist of path-completions.
 If NO-CONFIRM, assume that the user does not want to modify the initial prompt."
   (interactive)
-  (let* ((node (org-roam-node-read initial-input filter-fn))
-         (node-meta (get-text-property 0 'meta node)))
-    (if node-meta                          ; node exists
+  (let ((node (org-roam-node-read initial-input filter-fn)))
+    (if (org-roam-node-file node)
         (progn
-          (find-file (plist-get node-meta :path))
-          (goto-char (plist-get node-meta :point)))
-      (let ((org-roam-capture--info `((title . ,node)
+          (find-file (org-roam-node-file node))
+          (goto-char (org-roam-node-point node)))
+      (let ((org-roam-capture--info `((title . (org-roam-node-title node))
                                       (slug  . ,(funcall org-roam-title-to-slug-function node))))
             (org-roam-capture--context 'title))
         (setq org-roam-capture-additional-template-props (list :finalize 'find-file))
         (org-roam-capture--capture)))))
 
+(defun org-roam-node-insert (&optional completionsn filter-fn)
```

Comment:
Potential typo here on `completionsn`?

Just trying to help. 😇

Reply:
Fixed, thanks!